chrome/locale/sv-SE/quicktext.properties

```diff
@@ -11,9 +11,9 @@ template=Mall
 newGroup=Ny grupp
 newTemplate=Ny mall
 newScript=Nytt skript
-remove=Är du säker på att du vill tabort "%S"?
+remove=Är du säker på att du vill tabort ”%S”?
```

Comment:
the same for opening and closing quote?

Reply:
https://en.wikipedia.org/wiki/Quotation_mark#Summary_table

Reply:
Thanks for the link.